pandas/io/stata.py

```diff
@@ -497,29 +495,10 @@ class ValueLabelTypeMismatch(Warning):
     pass
 
 
-value_label_mismatch_doc = """
-Stata value labels (pandas categories) must be strings. Column {0} contains
-non-string labels which will be converted to strings.  Please check that the
-Stata data file created has not lost information due to duplicate labels.
-"""
-
-
 class InvalidColumnName(Warning):
     pass
 
 
-invalid_name_doc = """
-Not all pandas column names were valid Stata variable names.
-The following replacements have been made:
-
-    {0}
-
-If this is not what you expect, please make sure you have Stata-compliant
-column names in your DataFrame (strings only, max 32 characters, only
-alphanumerics and underscores, no Stata reserved words)
-"""
```

Comment:
Same comment here.

Reply:
Agreed again.  Though if you want to change `{0}` to use a named argument, id be on board with that